stylesheet/main.css

```diff
@@ -57,6 +58,24 @@ body {
     color: #00a9e2;
 }
 
+/* The rocket image won't display unless it is in a div */
+.rocketContainer {
+    position: absolute;
+    bottom: 15px;
+    right: 10px;
+    width: 200px;
+    height: 250px;
+}
```

Comment:
Also, when you change position to relative, you also want to remove the `bottom` and `right` props

Reply:
I'll give it a go

Reply:
I've done these changes, check and let me know. Bev is going to be adding a longer rocket that I will replace this one once it's done